mscore/seq.cpp

```diff
@@ -1750,7 +1750,15 @@ void Seq::setLoopSelection()
 
       if (score && score->selection().isRange()) {
             cs->setLoopInTick(score->selection().tickStart());
-            cs->setLoopOutTick(score->selection().tickEnd());
+            cs->setLoopOutTick(score->selection().tickEnd());      
+      }
```

Comment:
The code style rules (current ones, at least - they may be changing soon) require close braces to be indented once.